user-tools/README-CV Object Detection.md

```diff
@@ -0,0 +1,150 @@
+## Bot Segment Type
+
+###  CVObjectDetection
+
+Disclaimer: The `CVObjectDetection` type is still in an experimental phase and may provide inconsistent or unexpected results in many situations.  We are continuing to evaluate and tune this type.  See the [Limitations and Notes](#limitations-and-notes) section below for more information.
+  - The `CVObjectDetection` type can be used in Bot Segments for both `endCriteria` and/or `botAction`.  This type looks for the presence of an object in the current frame of the game until a frame contains the object. The object can either be specified by an image or by text.  This type communicates with our remote AIService instance to perform the evaluation.  When a result is found, the AIService provides the bounding rect information to our SDK to confirm the existence and location of the supplied image in the current frame.
+
+**TODO:** [Loom Video - Feel free to ignore this as I will be making these from my own demos]
+
+#### **CVObjectDetection via textQuery**
+
+When a textQuery is specified the AIService will search for the specified object in the image. For example, if the textQuery is "Dog" then the AIService will look for a dog in the image. If a dog is detected then the AIService will return the bounding box of the dog.
+
+This approach is useful when you want to detect an object that can look differently in the game. For example, you might want to click on a "Tree" but there might be several types of trees in the game, and you do not want to differentiate between them. If you know exactly what the tree might look like (such as a static sprite) then [CV Image](user-tools\README-CVImage.md) might be a better option for you.
```

Comment:
It might not just be that the tree looks differently itself, I would say this is useful when the orientation of the object in the camera view is dynamic, and it can't match directly to a static image.